src/web/lib/braze/initialiseAppboy.ts

```diff
@@ -0,0 +1,19 @@
+const SDK_OPTIONS = {
+	enableLogging: false,
+	noCookies: true,
+	baseUrl: 'https://sdk.fra-01.braze.eu/api/v3',
+	sessionTimeoutInSeconds: 1,
+	minimumIntervalBetweenTriggerActionsInSeconds: 0,
+};
+
+const getInitialisedAppboy = async (apiKey: string): Promise<typeof appboy> => {
```

Comment:
Is this memoized as written (and do we want it to be)? It looks like if the Epic and Banner were both to request this, we'd do the import and initialisation code twice?

Reply:
Good spot, no it's not memoized at this point. The plan is to do that at the point where we need it.